### ISSUE TITLE
Accessibility: Checkout payment and shipping forms - missing fieldsets

### ISSUE THREAD

**Issue by msiewierska** [shopware/platform#13980]
### Shopware Version

6.7.3.1

### Affected area / extension

Platform(Default)

### Actual behaviour

For payment options and shipping options in Checkout step 2 there are missing fieldsets:

A fieldset provides a visual and structural grouping of related form elements. It is typically necessary for groups of check boxes or radio buttons where a higher level description (called a legend) is necessary to understand the function of the check boxes or radio buttons.

![Image](https://github.com/user-attachments/assets/15261b8c-e31a-48e3-9509-5fc65533b26c)

### Expected behaviour

Fieldsets should be added to ensure compliance with success criterion 1.3.1 (level A).

### How to reproduce

Add product to the cart, go the the checkout -  confirm step. Inspect payment and shipping forms.
